### ISSUE TITLE
There seems to be a problem with the MobilNetV3 structure

### ISSUE THREAD

**Issue by Daming-TF** [Deci-AI/super-gradients#365]
Hello, the network structure of MobileNetV3 seems to be different from that of the original author. The SE module of V3 is not a module that uses SENet. The author changed nn.Line to nn.Conv

**Comment by oferbaratz** [comment]
Hey @Daming-TF
Thanks for the issue, we are opening a bug and it will be fixed soon.

**Comment by lkdci** [comment]
Hi @Daming-TF , there are some Squeeze & Excite versions which uses Conv op insteaf of Linear. 

Conv and Linear are equivalent when applied upon an input tensor with only one neuron per channels, which is the output after the Global average pooling operator. (BxCx1x1 or simply BxC). 

In both cases the layers num MAC is represented by Cin x Cout.

There are indeed some hardware accelerators which prefer the Convolution instead of the fully connected one, so in some cases the convolution operation might run faster.

**Comment by Daming-TF** [comment]
Hello@lkdci oh  that's true😄, thanks I got it
